SerialPrograms/Source/PokemonSwSh/Programs/DateSpamFarmers/PokemonSwSh_DateSpam-PokeJobsFarmer.cpp

```diff
@@ -0,0 +1,158 @@
+/*  Poke Jobs Farmer
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#include "Common/Cpp/PrettyPrint.h"
+#include "NintendoSwitch/Commands/NintendoSwitch_Device.h"
+#include "NintendoSwitch/Commands/NintendoSwitch_PushButtons.h"
+#include "PokemonSwSh/PokemonSwSh_Settings.h"
+#include "PokemonSwSh/Commands/PokemonSwSh_Commands_DateSpam.h"
+#include "PokemonSwSh_DateSpam-PokeJobsFarmer.h"
+
+namespace PokemonAutomation
+{
+	namespace NintendoSwitch
+	{
+		namespace PokemonSwSh
+		{
+
+			PokeJobsFarmer_Descriptor::PokeJobsFarmer_Descriptor()
+				: RunnableSwitchProgramDescriptor(
+					  "PokemonSwSh:PokeJobsFarmer",
+                      STRING_POKEMON + " SwSh",
+                      "Date Spam - Poke Jobs Farmer",
+					  "ComputerControl/blob/master/Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md",
+					  "Farm Poke Jobs.",
+					  FeedbackType::NONE,
+					  PABotBaseLevel::PABOTBASE_12KB)
+			{
+			}
+
+			PokeJobsFarmer::PokeJobsFarmer(const PokeJobsFarmer_Descriptor &descriptor)
+				: SingleSwitchProgramInstance(descriptor),
+				  SKIPS(
+					  "<b>Number of days:</b>",
+					  200),
+				  CONCURRENCY(
+					  "<b>Number of concurrent Poke Jobs per day:</b>",
+					  2),
+				  MENU_INDEX(
+					  "<b>Index of Poke Jobs in Rotom menu:</b>",
+					  3),
+    			  m_advanced_options(
+    			      "<font size=4><b>Advanced Options:</b> You should not need to touch anything below here.</font>"),
+				  MASH_B_DURATION(
+					  "<b>Mash B for this long upon completion of Poké Job:</b>",
```

Comment:
Don't use the unicode é since we want to keep all source files purely ASCII. Use `QChar(0xe9)` instead.